src/Uno.Extensions.Configuration/ConfigurationBinder.cs

```diff
@@ -0,0 +1,731 @@
+﻿
+
+using System.Collections.Immutable;
+using System.ComponentModel;
+
+namespace Uno.Extensions.Configuration.Internal;
+
+public static class ConfigurationBinder
```

Comment:
Indeed LOT of reflection here ... would be great to replace that by `System.Text.Json` (in an other PR)

Reply:
Yeh, there's a trade off between converting configuration section to a string and then parsing using System.Text.Json versus the current logic that uses reflection. Loading configuration isn't done that frequently, and the conversion to strongly typed entity (ie this binder logic) isn't done immediately on startup, only when the entities are actually requested